copter/source/docs/traditional-helicopter-connecting-apm.rst

```diff
@@ -10,63 +10,54 @@ Autopilot Info
 
 .. image:: ../images/PH21_2.jpg
     :target: ../_images/PH21_2.jpg
 
-Motor 1, which is normally the left front servo on your helicopter goes to pin 1.
-Motor 2, which is normally the right front servo goes to pin 2.  Motor 3, which 
-is normally the rear (elevator) servo goes to pin 3. Tail servo goes to pin 4.
+The :ref:`autopilot output functions wiki <common-rcoutput-mapping>` shows the complete list of servo output functions.  The default swashplate is H3-120 where Motor 1, left front servo, goes to pin 1; Motor 2, right front servo goes to pin 2; and Motor 3, rear (elevator) servo, goes to pin 3. See the :ref:`swashplate setup wiki <traditional-helicopter-swashplate-setup>` for more details.
 
-If you are using a DDVP (Direct Drive Variable Pitch) tail rotor, the tail motor
-ESC connects to pin 7. The throttle servo or ESC for the main rotor motor
-connects to pin 8.
+Tail servo is designated as Motor 4 and is defaulted to pin 4. Direct Drive Fixed Pitch (DDFP) tail rotors will also be connected to Motor 4 and the tail type parameter (:ref:`H_TAIL_TYPE <H_TAIL_TYPE>`) set to DDFP.  Direct Drive Variable Pitch (DDVP) tail rotors will use Motor 4 (defaulted to pin 4) to control tail rotor pitch and the tail ESC connection is defaulted to pin 7. This is automatically configured as tail RSC for the servo 7 function when the tail type parameter (:ref:`H_TAIL_TYPE <H_TAIL_TYPE>`) is set to DDVP. 
+ 
+The throttle servo or ESC for the main rotor motor is defaulted to pin 8.  This is automatically configured as Heli RSC for the servo 8 function. See the :ref:`rotor speed control setup wiki <traditional-helicopter-rsc-setup>` for more details on RSC setup.  All traditional helicopter frames are required to use Motor Interlock.  This feature adds an extra layer of safety when working with helicopters.  Motor interlock enables the motor to drive the rotor/tailrotor.  This is similar to throttle hold in RC helicopters.  Motor interlock enabled (throttle hold off) means the motor is allowed to drive the rotor/tailrotor and the rotor speed control handles the rotor runup/shutdown.  Motor interlock disabled (throttle hold on) means the motor is not allowed to drive the rotor/tailrotor.  In order to arm the helicopter, the motor interlock must be disabled (throttle hold on). In ArduCopter 3.6 and earlier, the motor interlock and RC passthrough mode is tied to RC channel 8 only.  The RC transmitter channel 8 must have the PWM within 10 pwm of the RC8_MIN for motor interlock disabled (throttle hold on).  All other PWM values will set motor interlock enabled (throttle hold off).  In ArduCopter 4.0, the RCn_Option parameter can be set to motor interlock for a user selectable channel.  The transmitter channel on which the motor interlock is set requires the PWM to be low (<1200 PWM) for motor interlock disabled (throttle hold on) and above 1200 PWM for motor interlock enabled (throttle hold off).  
```

Comment:
I have tested this, but is there any limitation on using, say Channel 5 or 6, for throttle control in 4.0 when those channels are normally pre-set for flight modes and tuning? I know those can be re-assigned as well. But what setup problems occur if the throttle is assigned to 5, when 5 is also pre-assigned to flight modes? Does the system spit out an error message for that?

Reply:
easy to test in SITL I think, assuming there is a trad heli sitl model...
Chris, can I call you or skype you to discuss some things?